composer.json

```diff
@@ -5,7 +5,8 @@
     "require": {
     },
     "require-dev": {
-        "pds/skeleton": "~1.0"
+        "pds/skeleton": "~1.0",
+        "phing/phing": "2.16.0"
```

Comment:
After I added this line for phing, composer complained about being out of sync with the lock file and suggested that I run `composer update`. I did. The diff was big. Is the huge diff for the lock file expected?